### PR TITLE
Expanded Image Modal + Cleaned up console.log littering

### PR DESCRIPTION
**Pull request by Daniel-Ghaly**
Clicking on product image brings up expanded image modal. Still working on zoom-in functionality that should zoom image modal upon click.

Fixed issue with 'key' and 'className' React errors littering our console log.

### CODE REVIEW

**Comment by FlightfulKiwi** [comment]
Really good all around. Not much to comment on. Will approve after Ryan's has been merged.

**Review by FlightfulKiwi — APPROVED** [review]

**Comment by ryanaolea** [comment]
yall up late. Get your sleeeep!

**Pull request merged**
